model/src/test/java/com/javahelp/model/userEntityTest/UserEntityTest.java

```diff
@@ -0,0 +1,76 @@
+package com.javahelp.model.userEntityTest;
+
+import com.javahelp.model.user.UserInfo;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.UserType;
+import com.javahelp.model.user.ProviderUserInfo;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+public class UserEntityTest {
+
+    /** These are tests for User Entity
+     *
+     * --- Attributes ----
+     * user1 - The first user object
+     * user2 - the second user object
+     * userinfo1 - the information of the first user
+     * userinfo2 - the information of the second user
+     */
```

Comment:
Reiterating the point from `User`, the convention with Javadocs is to put them above the class, with no whitespace between the bottom of the comment, and the start of the class. Additionally, the conventional way to add Javadocs for fields is to place separate comments above the fields. For private fields, this is not necessary.